test/sema/interface_nested_into_subroutine.f90

```diff
@@ -39,3 +39,4 @@ subroutine sub
 end subroutine
 
 end program
+
```

Comment:
newline character (mandatory) at the end of file is not the same thing as empty line at the end of file (not recommended).